docs/ota-client-guide/modules/ROOT/pages/upload-large-binary.adoc

```diff
@@ -48,8 +48,10 @@ garage-sign targets upload \
   --name <name-of-target> \
   --version <target-version>
 ----
++
+TIP: You can find the `name` and `version` fields in the `targets.json` file under the `custom` field, as shown in the xref:customise-targets-metadata.adoc#_anatomy_of_targets_json_metadata[example].
 
-. If the upload is successful, in the `targets.json` file, specify the following metadata about the binary file: name, version, and types of hardware for which your binary file is intended, as shown in the xref:customise-targets-metadata.adoc#_anatomy_of_targets_json_metadata[example].
+. If the upload is successful, in the `targets.json` file, specify the binary metadata, as shown in the xref:customise-targets-metadata.adoc#_anatomy_of_targets_json_metadata[example].

```

Comment:
Two comments here:

* I know you didn't introduce this, but I think that linking to the example metadata this way might be more distracting than helpful--the command we specify generates the metadata so that the user doesn't have to worry about manually editing `targets.json`. If you think the link is valuable, I'd propose moving it to a footnote.

```suggestion
. If the upload is successful, add it to your local Targets metadata. The name and version you specify here must be the same as the ones you gave in the upload step.

```

Reply:
Dropped this info and modified the step.